src/Sylius/Behat/Context/Api/EmailContext.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Core\Test\Services\EmailCheckerInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Webmozart\Assert\Assert;
+
+class EmailContext implements Context
```

Comment:
Don't we have any email context classes already?

Reply:
We have but they are UI contexts :/

Reply:
I will leave this in this PR as it is and will make a separate PR with changes. I will extract EmailContext and make this changes so the sylius code would be a better place for everyone 🌈